main.nf

```diff
@@ -44,6 +51,51 @@ def helpMessage() {
     """.stripIndent()
 }
 
+
+//set directories
+pwd_path = "/Users/bichmann/Projects/Replicate_Workflow/NextFlow"
+openms_path = "/Users/bichmann/Programs/OpenMS/OpenMS_build/bin"
+results_path = "${pwd_path}/results"
+comet_executable_path = "/Users/bichmann/Programs/OpenMS/OpenMS/THIRDPARTY/MacOS/64bit/Comet/comet.exe"
+percolator_executable_path = "/Users/bichmann/Programs/OpenMS/OpenMS/THIRDPARTY/MacOS/64bit/Percolator/percolator"
+
+
+
+// fasta database
+params.fasta = "${pwd_path}/HUMAN_2014.fasta"
+fastafile = file(params.fasta)
+
+
+
+// mzml files
+params.mzmlfiles = "${pwd_path}/*.mzML"
```

Comment:
Set up a channel for the input files and then check with `ifEmpty()` for presence of file(s): 
cf https://github.com/nf-core/rnaseq/blob/1cd5ab733a35727b01370de2ae8462d65f57fbb9/main.nf#L185